msal/src/Microsoft.Identity.Client/Features/PublicClientWithTokenCache/PublicClientApplication.cs

```diff
@@ -61,5 +66,18 @@ public PublicClientApplication(string clientId, string authority, TokenCache use
         {
             UserTokenCache = userTokenCache;
         }
+
+
+        private static void GuardOnMobilePlatforms()
+        {
+#if ANDROID || iOS
+        throw new PlatformNotSupportedException("You should not use this constructor that takes in a TokenCache object on mobile platforms. " +
```

Comment:
Pretty sure I'm missing a few commas :). @jmprieur and @jennyf19 - can you take a look ?

Reply:
looks good.

---
In reply to: [237924601](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1444#discussion_r237924601) [](ancestors = 237924601)